bin/docserv-createconfig

```diff
@@ -229,18 +229,52 @@ def get_git_branches(path):
             result[branch_wo_origin] = []
     return result
 
-def get_args(args):
+def check_args(args):
     """Checks if no argumets were set when running the script and asks for them.
-    If no languages were given, 'en_us' is set by default.
 
     :param args: The namespace containing the scripts arguments
     :type args: :class:`argparse.Namespace`
     """
-    while not args.path:
+    diff_checker = 1
+    default_checker = 1
+    languages = ("en-us de-de fr-fr pt-br ja-jp zh-cn es-es "
+                      "it-it ko-kr hu-hu zh-tw cs-cz ar-ar pl-pl "
+                      "ru-ru").split(" ")
+    # checking path
+    while not args.path or not os.path.exists(args.path):
         args.path = input("# Path to the git repository: ")
-    if not args.langs:
-        user_input = input("# Languages (pressing return sets 'en_us'): ")
-        args.langs = user_input if user_input else "en-us"
+    # checking languages
+    while diff_checker:
+        if not args.langs:
+            args.langs = input("# Languages: ")
```

Comment:
According to the next line, languages seems to be separated by comma. Therefor, I'd suggest:

```suggestion
            args.langs = input("# Languages (separated by comma): ")
```